pkg/types/types.go

```diff
@@ -28,3 +28,10 @@ const (
 	InstanceTypeEngine  = "engine"
 	InstanceTypeReplica = "replica"
 )
+
+const (
+	GlobalMountPathPattern = "/host/var/lib/kubelet/plugins/kubernetes.io/csi/driver.longhorn.io/*/globalmount"
+	VolumeGlobalMountPath  = "/host/var/lib/kubelet/plugins/kubernetes.io/csi/driver.longhorn.io/%x/globalmount"
```

Comment:
Could you point to where `VolumeGlobalMountPath` is used?

Reply:
It is not used in the new implementation, removed, thanks!